packages/icon/package.py

```diff
@@ -27,6 +27,14 @@ class Icon(AutotoolsPackage):
             git='ssh://git@gitlab.dkrz.de/icon/icon-nwp.git',
             submodules=True)
 
+    # Optional Features:
+    variant(
```

Comment:
Aren't r-paths automatically injected by the  Spack compiler wrappers?

I think this is the reason why it is disabled for icon configure.

And looking at the .so libs in icon.exe tey point to the correct libs, even when used outside of spack environment.
And this can only work when r-paths are already used.

Reply:
Aha, that makes a lot of sense. Then this variant is of course redundant. I'll change it back.